src/Classifier.php

```diff
@@ -44,17 +44,11 @@ public function classify(Client $parseResult) : Result
                 continue;
             }
 
-            $result->isMobileDevice = in_array($rule['class'], ['tablet', 'phone']);
-            $result->isPhone        = $rule['class'] === 'phone';
-            $result->isTablet       = $rule['class'] === 'tablet';
-            $result->isSpider       = $rule['class'] === 'spider';
-            $result->isComputer     = $rule['class'] === 'desktop';
-            $result->matchType      = $rule['class'];
-
+            $matchClass = $rule['class'];
             break;
         }
 
-        return $result;
+        return new Result($matchClass);
```

Comment:
And perhaps this could be an exception? Or is a match class with an empty string valid?